lib/product_list/widgets/search_results_header.dart

```diff
@@ -0,0 +1,43 @@
+import 'package:flex_storefront/search/models/search_results.dart';
+import 'package:flutter/material.dart';
+
+class SearchResultsHeader extends StatelessWidget {
```

Comment:
Is this widget too high-level to be part of flex-ui?

If not, (i.e. it can be part of flex-ui), remove the Padding widget, because it's related to where the widget is used, not the widget itself.

Reply:
It can be included in Flex UI, but will need more properties added with time to make it adaptable per use case.

Reply:
Yep 👍 that would be the case with pretty much every widget in flex-ui, they will mature over time. We are still adding properties to the Costco ones